src/libdrakvuf/vmi.c

```diff
@@ -306,6 +306,12 @@ event_response_t pre_mem_cb(vmi_instance_t vmi, vmi_event_t* event)
     drakvuf_t drakvuf = event->data;
     drakvuf->regs[event->vcpu_id] = event->x86_regs;
 
+    if (event->mem_event.gfn == drakvuf->zero_page_gfn)
+    {
+        PRINT_DEBUG("Somebody try to do something to the empty page, let's emulate it\n");
```

Comment:
Since `pass` is not being used in this case, it should be freed to solve the Travis error.